test/features/steps/utils.py

```diff
@@ -0,0 +1,18 @@
+import os
+from crmsh import utils, bootstrap
+
+def run_command(cmd, stdout=False, stderr=False):
+    rc, out, err = utils.get_stdout_stderr(cmd)
+    if out and stdout:
+        print(out)
+    if err and stderr:
+        print(err)
+    return rc, out, err
+
+def check_cluster_status(nodelist):
+    if nodelist == "local":
```

Comment:
Why are we using this `local` variable. All the tests are executed in local so far